### ISSUE TITLE
Validation split argument

### ISSUE THREAD

**Issue by micah5** [aws/sagemaker-python-sdk#2637]
**Describe the feature you'd like**
An input argument to `fit()` that automatically splits the training data into train & validation sets.

**How would this feature be used? Please describe.**

Input float between 0 and 1 to fraction of the training data to be used as validation data. 

i.e. instead of
```python
model.fit(inputs={'train': s3_input("s3://.../train"), 'validation': s3_input("s3://.../valid")}) 
```

it would be something like:
```python
model.fit(inputs={'train': s3_input("s3://.../train"), 'validation_split': 0.2}) 
```

and 20% of the training data would be held away as a validation set.

**Describe alternatives you've considered**
See the implementation in keras: https://keras.rstudio.com/reference/fit.html

**Comment by akrishna1995** [comment]
We have added your feature request to our backlog of feature requests and may consider putting it into future SDK versions. I will go ahead and close the issue now, please let me know if you have any more feedback.
